integration-libs/cdc/organization/user-registration/cdc-b2b-register.module.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { NgSelectModule } from '@ng-select/ng-select';
+import { I18nModule, UrlModule } from '@spartacus/core';
+import {
+  FormErrorsModule,
+  NgSelectA11yModule,
+  SpinnerModule,
+} from '@spartacus/storefront';
+import { UserRegistrationFormService } from '@spartacus/organization/user-registration/components';
+import { CDCB2BRegisterComponentService } from './cdc-b2b-register-component.service';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    RouterModule,
+    UrlModule,
+    I18nModule,
+    SpinnerModule,
+    FormErrorsModule,
+    NgSelectModule,
+    NgSelectA11yModule,
+  ],
+  providers: [
+    {
+      provide: UserRegistrationFormService,
+      useClass: CDCB2BRegisterComponentService,
```

Comment:
For consistency and better developer experience of customers, I suggest that the extending service should use the name of the original original services + just add a custom prefix. For instance:
```ts
provide:     UserRegistrationFormService,
useClass: CdcUserRegistrationFormService
```